index.js

```diff
@@ -58,7 +58,17 @@ var handlers = {
         this.emit('AMAZON.StopIntent');
     },
     'AMAZON.HelpIntent': function () {
-        this.emit('Unhandled');
+        var self = this;
+        ApiAi.eventRequest({name: 'HELP'}, {sessionId: alexaSessionId})
+            .on('response', function (response) {
+                var speech = response.result.fulfillment.speech;
+                self.emit(':ask', speech, speech);
+            })
+            .on('error', function (error) {
+                console.error(error.message);
+                self.emit(':tell', error.message);
+            })
+            .end();
     },
```

Comment:
We might want to refactor the code and put these event handlers in separate functions. This is starting to look like a lot of repeat code.